src/pkg/build/builder_remote.go

```diff
@@ -99,6 +99,11 @@ func (rb *RemoteBuilder) Build(ctx context.Context) (err error) {
 		return err
 	}
 
+	fmt.Printf("\t+-> Build job submited with ID:\t%v\n", rd.ID.Hex())
+
+	fmt.Printf("Once your build has completed, your image will be available at %v\nOr you can pull it with the CLI by running\nsingularity pull --library-uri %v library://%v/build/image",
```

Comment:
I'd be tempted to adjust the newlines here. With the text + first URL (available at...) it will be way over 80 chars. Might be nicer to have the URL on a newline, indented or something like that?